docs/process.md

```diff
@@ -71,6 +72,12 @@ After writing the article, Share the GitHub pull request link to your article fo
 
 This gives community members and other writers the chance to have a read of the article and add constructive criticism. Once reviewed, the author makes the necessary changes ready for publishing. This might need two rounds of review if the edit is considerable.
 
+**Pre-review checks**
+* The first reviewer should copy and paste the article into a tool that verifies for plagiarism and AI-generated content. One tool for this check is https://www.zerogpt.com/
```

Comment:
Add blank lines before bullet lists. Markdown will be valid without it, but I find it easier to navigate with extra blank lines before and after each bullet list or heading.

Reply:
Done 👍